### ISSUE TITLE
 Won't Find Root If React Mounted Asynchronously

### ISSUE THREAD

**Issue by lcharbon** [webdriverio/webdriverio#3916]
**Environment (please complete the following information):**
 - **WebdriverIO version:** e.g. 5.8.0
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** v10.15.3
 - **NPM version:** 6.4.1
 - **Browser name and version:**  Chrome  74
 - **Platform name and version:** MacOS 10.14.4
 - **Additional wdio packages used (if applicable):**

**Config of WebdriverIO**
An example of how you configured WebdriverIO config
````
exports.config = {
    runner: 'local',
    port: 9515,
    path: '/',
    chromeDriverArgs: [ '--port=9515', '--url-base=\'/\'' ],
    chromeDriverLogs: './',
    specs: [ './tests/spec/**/*.js' ],
    exclude: [],
    maxInstances: 1,
    capabilities: [ { maxInstances: 1, browserName: 'chrome' } ],
    logLevel: 'silent',
    bail: 0,
    baseUrl: 'http://localhost:3000/',
    waitforTimeout: 10000,
    connectionRetryTimeout: 90000,
    connectionRetryCount: 3,
    services: [ 'chromedriver' ],
    framework: 'mocha',
    reporters: [ 'spec' ],
    mochaOpts: { ui: 'bdd', timeout: 60000 },
    beforeTest: function (test) {
        const chai = require('chai');
        chai.use(require('chai-string'));
        chai.use(require('chai-webdriverio').default(browser));
        global.assert = chai.assert;
        global.expect = chai.expect;
    }
}
````

**Describe the bug**
If react is mounted asynchronously.  Using `react$` selector throws the error "javascript error: Could not find the root element of your application".

**To Reproduce**
Clone: https://github.com/lcharbon/webdriverio_bugs_demo
Run `yarn install` command.
Use @wdio/cli to configure wdio as per the above settings.
Run:  `yarn test` command.

**Expected behavior**
Webdriverio to wait to find root component even if react is mounted asynchronously.

**Additional context**
In index.js of the cloned repository removing the `setTimeout()` wrapper will result in react mounting synchronously and the test running without error.

**Comment by baruchvlz** [comment]
Why would you mount React asynchronously?

**Comment by christian-bromann** [comment]
Closing as this seems to got fixed in resq 1.5.0 which was updated in WebdriverIO
